README.md

```diff
@@ -1,2 +1,31 @@
-# scriptorium
-A collection of scripts
+# Scriptorium
+
+This PlanetaryPy repository is not a focused library, but a collection
+of scripts.  You may find something here useful, and if so, feel
```

Comment:
maybe add a bit more here, e.g.: "scripts useful for Planetary Scientists." And move the concept paragraph up. I think the download instructions could be moved down to their own subsection

Reply:
So I think there will be many more folks coming to "use" a script than there will be who will "deposit" a script, so I wanted to have the "download" instructions close to the top.  I've broken things up into sections, but did leave the "Use a Script" instructions at the top.